admin/base/urls.py

```diff
@@ -10,7 +12,9 @@
     url(base_pattern,
         include(patterns('',
                          url(r'^$', views.root),
+                         url(r'^django_admin/', include(admin.site.urls)),
```

Comment:
Is this a way we can obfuscate the admin login? In the future this might look like: `admin.osf.io/admin` but with potentially obfuscated names.